ember/app/ui/cases/detail/index/controller.js

```diff
@@ -228,10 +228,12 @@ export default class CasesDetailIndexController extends Controller {
       }
 
       const blob = yield response.blob();
-      const filename = `${this.intl.t("documents.accountingExportFilename", {
-        dossierNo:
-          this.case.document.answers.edges.firstObject.node.StringAnswerValue,
-      })}`;
+
+      const filename = decodeURIComponent(
+        response.headers
+          .get("content-disposition")
+          .match(/filename\*?=['"]?(?:UTF-\d['"]*)?([^;\r\n"']*)['"]?;?/i)[1]
+      );
```

Comment:
This is probably the same function, as we use for the case downloads. In python I'd put it in a top-level `utils.py` (or similar).

Reply:
Extracted the download